storage/api/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogMetadataManager.java

```diff
@@ -196,7 +196,7 @@ void onPartitionLeadershipChanges(Set<TopicIdPartition> leaderPartitions,
 
     /**
      * This method is invoked only when the topic partitions are stopped on this broker. This can happen when a
-     * partition is emigrated to other broker or a partition is deleted.
+     * partition is immigrated to other broker or a partition is deleted.
```

Comment:
is this really a typo? Don't both mean the same thing (one is american english and other is british english)?

Reply:
Yes I do not think this is a typo, both of them are correct and true, this pull request can be closed